docs/api/app.md

```diff
@@ -647,6 +647,17 @@ Sets the application's [dock menu][dock-menu].
 
 Sets the `image` associated with this dock icon.
 
+### `app.unityLauncher.setBadgeCount(count)` _Ubuntu Unity_
```

Comment:
`Ubuntu Unity` is not a platform, we should say `_Linux_`, and make it clear in the docs that it only works for Unity launcher.